### ISSUE TITLE
Add map support

### ISSUE THREAD

**Issue by chris579** [gruppe-adler/grad_trenches#25]
Sticky issue for collecting map requests

* [ ] Da Kong [Unsung](https://www.armanam.eu/terrain.html)
* [ ] Doung [Unsung](https://www.armanam.eu/terrain.html)

**Comment by TheNightstalk3r** [comment]
G.O.S Maps like Al Rayak, Gunkizli etc?

**Comment by Rockfrog** [comment]
If you're taking requests (if not, ignore me!), it'd be great to add the following support if you could?

1. [Bozcaada](http://steamcommunity.com/sharedfiles/filedetails/?id=524622628)
2. [G.O.S N'Djenahoud](http://steamcommunity.com/sharedfiles/filedetails/?id=937515516)
3. [Kidal](http://steamcommunity.com/sharedfiles/filedetails/?id=696334883)

A very nice mod you've put together here.

**Comment by chris579** [comment]
`Al Rayak`, `Leskovets` and `Kerama` will be included in the next release.
We're always taking requests @Rockfrog however we can't promise that we are processing them as it is a time intense task to add new maps. We have to download them, find the suitable paths and prefixes and have to do some testing to make sure our configs are fine. This takes a lot of time, especially if maps are only published on Armaholic and not on Steam Workshop as some moron thought it would be a good idea to limit the download speed there.

**Comment by Rockfrog** [comment]
No, absolutely understand, anything whenever it fits your schedule is welcomed. Mod Devs have their own lives and agendas - it's your time, and your mod!

Those links all go to Steam Workshop. Believe Bozcaada just uses Altis & Stratis ground, admittedly the other two are more custom, but they're both deserts, Kidal is very plain.

**Comment by byjokese** [comment]
[Hebontes map](https://steamcommunity.com/sharedfiles/filedetails/?id=1412917989).

**Comment by NortonDK** [comment]
Kujari

**Comment by Bob-Murphy** [comment]
Hey guys,
could you please consider adding Kunduz to the list?
You simply can take the same ones like for Anizay as it's the same type of map.
Thank you very much in advance and best regards,
Bob

**Comment by GoldJohnKing** [comment]
Hi, would you please add support for Virolahti (VT7) terrain? Thanks for your effort!

**Comment by AdlerSalbei** [comment]
> 
> 
> Hi, would you please add support for Virolahti (VT7) terrain? Thanks for your effort!

It is already supported
